ts/IterableInboxMessageDisplay.tsx

```diff
@@ -46,13 +49,40 @@ const IterableInboxMessageDisplay = ({
    returnButtonContainer = {...returnButtonContainer, marginTop: Platform.OS === 'android' ? 0 : 40}
    returnButtonContainer = (!isPortrait) ? {...returnButtonContainer, marginTop: 10} : returnButtonContainer
 
+   let JS = `
+      const links = document.querySelectorAll('a')
+      links.forEach(link => {
+         link.addEventListener("click", () => {
+            window.ReactNativeWebView.postMessage(link.href)
+         })
+      })
+   `
+
+   let action = new IterableAction("", "", "")
```

Comment:
Instead of passing empty strings, we wil have to create a IterableAction with based on what action we are going to perform:

Here is the link to Android SDK function of when we tap on a link on inapp.
https://github.com/Iterable/iterable-android-sdk/blob/3733809ee492f6fb45717a36df5ed6ec0e33a160/iterableapi/src/main/java/com/iterable/iterableapi/IterableInAppManager.java#L265

If it starts from iterable link, say action://, we define action type as action passed on to it(for e.g `delete`). Else, `openUrl` type.

Reply:
Per our huddle, I added a type and the url passed to the urlHandler as data to the IterableAction.